Add tauri-solid-ts-tailwind-vite-template (#108)
tauri-apps/awesome-tauri: README.md

- [tauri-react-parcel-template](https://github.com/henrhie/tauri-react-parcel-template) - React template with Parcel as build tool, TypeScript and hot module replacement.
- [tauri-rescript-template](https://github.com/JonasKruckenberg/tauri-rescript-template) - Tauri, ReScript, and React template.
- [tauri-sycamore-template](https://github.com/JonasKruckenberg/tauri-sycamore-template) - Tauri and Sycamore template.

> - [tauri-solid-ts-tailwind-vite-template](https://github.com/AR10Dev/tauri-solid-ts-tailwind-vite) - Solid template with Vite + HMR, TypeScript, Tailwind CSS, ESLint, Prettier, file based routing and cross-platform GitHub Actions builds.

Reviewer: ```suggestion
- [tauri-solid-ts-tailwind-vite-template](https://github.com/AR10Dev/tauri-solid-ts-tailwind-vite) - SolidJS Template preconfigured to use Vite, TypeScript, Tailwind CSS, ESLint and Prettier.
```

Adapted your repos README description to shorten this a bit
